### PR TITLE
support: add GSWIFT token icon

### PR DESCRIPTION
**Pull request by KaliszukTomasz**
### 📝 Description
Add logo for GSWIFT token to ledger. Support for tokens was added to ledger live desktop already
Contract address (arbitrum):
GSWIFT - 0x580e933d90091b9ce380740e3a4a39c67eb85b4c

### ❓ Context

- **JIRA or GitHub link**: https://github.com/KaliszukTomasz/ledger-live/tree/support/add-gswift-icon

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:**
  - 
---
 "@ledgerhq/crypto-icons-ui": major
 "@ledgerhq/live-common": major
 ---
---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/ledger-live-common/src/data/icons/svg/GSWIFT.svg
[the remark this answers is not in the document]
Comment (KaliszukTomasz) [reply libs/ledger-live-common/src/data/icons/svg/GSWIFT.svg:1]: please review

File: libs/ui/packages/crypto-icons/src/svg/GSWIFT.svg [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<path d="M8.6843 12.1396L5.90648 14.9184L3.89648 16.9284L6.51867 15.8315L12.0162 13.5328L10.6174 12.9478L8.52867 13.8215L9.76055 12.5896" fill="url(#paint0_linear_2616_140)"/>
+<path d="M14.5234 11.3156L14.9387 11.73L15.3484 12.1396L17.0762 11.4168L18.1525 10.9668L16.2522 10.5928L14.5234 11.3156Z" fill="url(#paint1_linear_2616_140)"/>
+<path d="M18.1255 14.9185L15.3477 12.1398L14.938 11.7301L14.5227 11.3157L13.7999 10.5929H12.2934H12.2352H9.92992L12.5099 11.672H13.3509L13.4118 11.7301L15.5034 13.8216L7.78117 10.5929L9.46492 8.90914H11.1055H12.0168H14.5677L16.2515 10.5929L18.1518 10.967L15.014 7.83008H13.4887H12.0168H11.1055H10.544H9.01773L7.93867 8.90914L6.70492 10.142L5.88086 10.967L6.95711 11.417L17.5134 15.8316L20.1355 16.9285L18.1255 14.9185Z" fill="white"/>
+</g>
+<defs>
+<linearGradient id="paint0_linear_2616_140" x1="6.17138" y1="16.7006" x2="9.86665" y2="13.2834" gradientUnits="userSpaceOnUse">
+<stop stop-color="#FFEE00"/>
+<stop offset="0.9878" stop-color="#FFA902"/>
+</linearGradient>
+<linearGradient id="paint1_linear_2616_140" x1="15.3568" y1="12.1392" x2="16.5365" y2="10.9395" gradientUnits="userSpaceOnUse">
```
Comment (gre) [inline libs/ui/packages/crypto-icons/src/svg/GSWIFT.svg:14]: we can't afford global id in the svg, nor "url()" they won't work when ported to React Native SVG components

**Review by gre — CHANGES_REQUESTED** [review]

File: libs/ui/packages/crypto-icons/src/svg/GSWIFT.svg (hunk 1 above)
Comment (KaliszukTomasz) [reply libs/ui/packages/crypto-icons/src/svg/GSWIFT.svg:14]: please check if the current svg version is ok

File: libs/ledger-live-common/src/data/icons/svg/GSWIFT.svg
Comment (gre) [inline libs/ledger-live-common/src/data/icons/svg/GSWIFT.svg:1]: sorry for missing this one: this file is no longer needed, we deduplicated our icons, the live-common/src/data folder is gone on develop branch.

Comment (KaliszukTomasz) [reply libs/ledger-live-common/src/data/icons/svg/GSWIFT.svg:1]: Ok, loaded new develop and moved svg to libs/ui/packages/crypto-icons/src/svg. Please check.

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**